cmd/manager/manager/expander.go

```diff
@@ -120,8 +121,26 @@ func (e *expander) expandConfiguration(conf *common.Configuration) (*ExpandedCon
 
 				// This was not a primitive resource, so add its properties to the layout
 				// Then add the all of the layout resources returned by the recursion to the layout
-				layout.Properties = resource.Properties
 				layout.Resources = expConf.Layout.Resources
+				layout.Properties = resource.Properties
+			} else {
+				// Raise an error if a non template chart supplies properties
+				if resource.Properties != nil {
+					return nil, fmt.Errorf("properties provided for non template chart %s", resource.Type)
+				}
+
+				additions = []*common.Resource{}
+				for _, member := range content.Members {
+					segments := strings.Split(member.Path, "/")
+					if len(segments) > 1 && segments[0] == "templates" {
+						if strings.HasSuffix(member.Path, "yaml") || strings.HasSuffix(member.Path, "json") {
+							resource, err := util.ParseKubernetesObject(member.Content)
```

Comment:
Why don't we parse it as an actual resource?  This means the yaml from the existing charts will need the "header" part with name added, but it's more consistent with the expandable charts, and means parameterizing a chart without properties can be done extremely easily, which will encourage users to write charts that are widely applicable.  It also means you can refer to a subchart here (filling in its parameters).  I think that will be a useful pattern.

Either way, I'd argue for propagating the error to the user, because a syntax error here is most likely to be a mistake, rather than intentional.